src/main/java/com/clussmanproductions/trafficcontrol/gui/TrafficLightControlBoxGui.java

```diff
@@ -216,8 +216,11 @@ public void initGui() {
 	private void setManualChecked()
 	{
 		greenOn.setIsChecked(getChecked(EnumTrafficLightBulbTypes.Green, false, true));
+		greenOn.setIsChecked(getChecked(EnumTrafficLightBulbTypes.SGreen, false, true));
```

Comment:
This is what I was saying in the other pull request. You don't need to do this because `greenOn` will always be whatever the result of this is. Same with all the others in this method.